esp/scm/ws_access.ecm

```diff
@@ -787,8 +787,8 @@ ESPresponse [nil_remove] UserAccountExportResponse
     [http_content("application/octet-stream")] binary Result;
 };
 
-
-ESPservice [requiredaccess("NONE"), version("1.09"), exceptions_inline("./smc_xslt/exceptions.xslt")] ws_access
+//Kevin/russ does this method need feature level check?
+ESPservice [autofeatureauth("NONE"), version("1.09"), exceptions_inline("./smc_xslt/exceptions.xslt")] ws_access
```

Comment:
@wangkx @RussWhitehead are we intentionally not requiring feature level auth on ws_access' methods?